exporter/prometheusremotewriteexporter/exporter_test.go

```diff
@@ -368,7 +368,7 @@ func runExportPipeline(ts *prompb.TimeSeries, endpoint *url.URL) error {
 		return err
 	}
 
-	return prwe.handleExport(context.Background(), testmap)
+	return prwe.handleExport(context.Background(), testmap, nil)
```

Comment:
Are there any tests that validate the configuration option is honored and metadata is converted and provided?

Reply:
I will add tests for it.